sdk/go/auto/local_workspace.go

```diff
@@ -160,10 +160,14 @@ func (l *LocalWorkspace) SetConfig(ctx context.Context, stackName string, key st
 	if val.Secret {
 		secretArg = "--secret"
 	}
+	pathArg := ""
```

Comment:
I'm not sure if this works, and if it does it will be fragile. If `!val.IsPath`, we will construct a command like:
```sh
pulumi config set 'foo[0]' --plaintext "" --stack dev -- bar
```
which will error since `""`  is not a known argument.

Since `path` is a flag, we can just set it to its value. 
```suggestion
        pathArg := fmt.Sprintf("--path=%t", val.IsPath)
```